parsec/backend/vlob.py

```diff
@@ -141,6 +181,68 @@ def __init__(
         self.sequester_services_certificates = sequester_services_certificates
 
 
+async def extract_sequestered_data_and_proceed_webhook(
+    services: Dict[SequesterServiceID, SequesterService],
+    organization_id: OrganizationID,
+    author: DeviceID,
+    encryption_revision: int,
+    vlob_id: VlobID,
+    timestamp: DateTime,
+    sequester_blob: Dict[SequesterServiceID, bytes],
+):
+    # Split storage services and webhook services
+    storage_service_ids = [
+        service.service_id
+        for service in services.values()
+        if service.service_type == SequesterServiceType.STORAGE
+    ]
+    webhook_service_ids = [
+        service.service_id
+        for service in services.values()
+        if service.service_type == SequesterServiceType.WEBHOOK
+    ]
+    for webhook_service_id in webhook_service_ids:
+        service = services[webhook_service_id]
+        sequester_data = sequester_blob[webhook_service_id]
+        if not service.webhook_url:
```

Comment:
This should never occur in theory given the service object comes from the database and has been validated when inserted.

However it's a good idea to keep this check, but I think we don't need a specific type of exception for this: just log an error (this time it's an error level log and not a warning given it shows an issue with our code) and just pretend the url was provided but didn't answer to the http request